### PR TITLE
Refactored away `LocalHistogramEffect` as a base class

### PR DESCRIPTION
**Pull request by Lehonti**

### CODE REVIEW

File: Pinta.Effects/Effects/LocalHistogram.cs [hunk 1]
```diff
@@ -33,12 +33,44 @@ private static void SetToZero (Span<int> data)
...
+		Span<int> hg,
+		Span<int> hr);
 
-	//same as Apply, except the histogram is alpha-weighted instead of keeping a separate alpha channel histogram.
-	public virtual ColorBgra ApplyWithAlpha (in ColorBgra src, int area, int sum, Span<int> hb, Span<int> hg, Span<int> hr)
-		=> src;
+	/// <summary>
+	/// Default implementation of <see cref="Applier"/>
+	/// </summary>
+	public static ColorBgra ReturnOriginal (
```
Comment (cameronwhite) [inline Pinta.Effects/Effects/LocalHistogram.cs:55]: Are these unused?

Comment (Lehonti) [reply Pinta.Effects/Effects/LocalHistogram.cs:55]: @cameronwhite yes, they are unused. They are there to provide placeholder methods for those who need it. I thought they would be needed for refactoring but that wasn't the case.

Comment (cameronwhite) [reply Pinta.Effects/Effects/LocalHistogram.cs:55]: Yeah, I think we should just remove them if none of the existing implementations needed them.

Comment (Lehonti) [reply Pinta.Effects/Effects/LocalHistogram.cs]: @cameronwhite I just removed them

**Pull request merged**